core/templates/core/base.html

```diff
@@ -59,14 +59,19 @@
 						<a class="nav-link" href="{% url 'achievements/all' %}">Achievements</a>
 					</li>
 					<li>
-						<a class="nav-link" href="{% url 'activity_calendar:calendar' %}">Activity Calendar</a>
+						<a class="nav-link" href="{% url 'activity_calendar:calendar' %}">Calendar</a>
 					</li>
                     <li>
                         <a class="nav-link" href="{% url 'boardgames:home' %}">Boardgames</a>
                     </li>
                     <li>
                         <a class="nav-link" href="{% url 'roleplaying:home' %}">Roleplay</a>
                     </li>
+                    {% if user.is_staff %} {# Temporary criteria while we create all commission data #}
```

Comment:
Why not add or use a permission instead? Those are easy to grant to all members or users in production without needing to update the code.